crates/node-core/src/args/network.rs

```diff
@@ -89,15 +89,15 @@ pub struct NetworkArgs {
     /// `GetPooledTransactions` request. Spec'd at 2 MiB.
     ///
     /// <https://github.com/ethereum/devp2p/blob/master/caps/eth.md#protocol-messages>.
-    #[arg(long = "pooled-tx-response-soft-limit", value_name = "BYTES", default_value_t = SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESPONSE, help = "Sets the soft limit for the byte size of pooled transactions response. Specified at 2 MiB by default. This is a spec'd value that should only be set for experimental purposes on a testnet.")]
+    #[arg(long = "pooled-tx-response-soft-limit", value_name = "BYTES", default_value_t = SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESPONSE, help = "Sets the soft limit for the byte size of pooled transactions response. Specified at 2 MiB by default. This is a spec'd value that should only be set for experimental purposes on a testnet.",long_help = None)]
     pub soft_limit_byte_size_pooled_transactions_response: usize,
 
     /// Default soft limit for the byte size of a `PooledTransactions` response on assembling a
     /// `GetPooledTransactions` request. This defaults to less
     /// than the [`SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESPONSE`], at 2 MiB, used when
     /// assembling a `PooledTransactions` response. Default
     /// is 128 KiB.
-    #[arg(long = "pooled-tx-pack-soft-limit", value_name = "BYTES", default_value_t = DEFAULT_SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESP_ON_PACK_GET_POOLED_TRANSACTIONS_REQ)]
+    #[arg(long = "pooled-tx-pack-soft-limit", value_name = "BYTES", default_value_t = DEFAULT_SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESP_ON_PACK_GET_POOLED_TRANSACTIONS_REQ,long_help = None)]
```

Comment:
we need some more user friendly description of what this does just like for `soft_limit_byte_size_pooled_transactions_response` which has a `help`

Reply:
added a user friendly explainer. Let me know if i have to add anything more